android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -174,10 +176,10 @@ public void setUpcomingSession() {
             upcomingSessionText.setText(upcomingTitle);
         } else {
             upcomingSessionTitle.setText(getResources().getString(R.string.no_upcoming_Sess));
-            upcomingSessionText.setVisibility(View.GONE);
+            upcoming_session_details.setVisibility(View.GONE);
         }
     }
-
+//===========================================================================================================
```

Comment:
What's this needed for?

Reply:
Actually sir I have added this to distinguish between the different portions of the code. Its for my refrnce. I should remove it before creating pull request  , my bad.